src/modules/user/model/wallet.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+  OneToOne,
+  JoinColumn,
+  Unique,
+} from "typeorm";
+import { User } from "./user.entity";
+
+@Entity()
+@Unique("UK_wallet_id_walletAddress", ["id", "walletAddress"])
+export class Wallet {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true })
+  userId: number;
+
+  @OneToOne(() => User)
+  @JoinColumn([{ name: "userId", referencedColumnName: "id" }])
+  user?: User;
```

Comment:
I think we also need to add some reference in the `user.entity.ts` right @amateima? Or `GET /users/wallets` in order for the FE to be able to know the linked wallet